core/src/main/java/com/linecorp/armeria/server/healthcheck/FixedRateHealthChecker.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the Licenses
+ */
+
+package com.linecorp.armeria.server.healthcheck;
+
+import java.time.Duration;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.function.Supplier;
+
+import com.linecorp.armeria.client.retry.Backoff;
+import com.linecorp.armeria.common.util.AbstractListenable;
+
+import io.netty.channel.EventLoop;
+
+/**
+ * Trigger supplied health checker after constructing and subsequently with the given interval.
+ */
+final class FixedRateHealthChecker extends AbstractListenable<HealthChecker>
+        implements ListenableHealthChecker {
+    private final AtomicBoolean isHealthy = new AtomicBoolean(false);
+    private final Supplier<CompletionStage<Boolean>> healthChecker;
+
+    FixedRateHealthChecker(Supplier<CompletionStage<Boolean>> healthChecker, Duration interval,
+                           double jitter, EventLoop eventLoop) {
+        this.healthChecker = healthChecker;
+
+        eventLoop.scheduleAtFixedRate(createTask(), 0,
+                                      Backoff.fixed(interval.toMillis())
+                                             .withJitter(jitter)
```

Comment:
I'm not sure if we need to take the `jitter` as an input.
Because `jitter` is used only once to calculate the delay, I think a user can specify the calculated interval using the `jitter`.
How about removing it? /cc @trustin

Reply:
If we have the random delay for the fixed rate as well, I think there's not so much difference with the `ofFixedDelay`.
How about removing `ofFixedRate` and just providing `ofFixedDelay` then?

Reply:
@kojilin Let's remove `ofFixedRate` and just provide `ofFixedDelay`.
Had a chat with @trustin and @ikhoon about it. There's a chance that the tasks are delayed and executed at once later which is undesirable.
`ofFixedDelay` can just cover the usage so let's only provide it. 😄